tests/e2e/utils/requestHandlers/CheApiRequestHandler.ts

```diff
@@ -8,14 +8,55 @@
  * SPDX-License-Identifier: EPL-2.0
  **********************************************************************/
 
-import axios, { AxiosResponse } from 'axios';
+import axios, { AxiosResponse, AxiosRequestConfig } from 'axios';
 import { TestConstants } from '../../TestConstants';
 import { TYPES } from '../../inversify.types';
 import { inject, injectable } from 'inversify';
 import { IAuthorizationHeaderHandler } from './headers/IAuthorizationHeaderHandler';
 
 @injectable()
 export class CheApiRequestHandler {
+
+    /**
+     * This method adds a request interceptor into axios request interceptors list and returns an ID of the interceptor
+     */
+    public static enableRequestInteceptor(): number {
+        console.log(`CheApiRequestHandler.enableRequestInterceptor`);
```

Comment:
It would be nice to use Logger instead of console.log here. I'm thinking about level DEBUG.

Reply:
And in all other places ;)

Reply:
I wanted to do that, trust me, but Logger was messing up echoing the object out while console.log has no issues.